projects/packages/videopress/src/class-access-control.php

```diff
@@ -179,6 +179,7 @@ private function check_block_level_access( $restriction_details, $guid, $embedde
 			$paywall             = \Automattic\Jetpack\Extensions\Premium_Content\subscription_service();
 
 			// Only paid subscribers should be granted access to the premium content.
+			$access_level = '';
```

Comment:
This is correctly a string, though going down a wrong rabbit hole I discovered the sister method `Abstract_Token_Subscription_Service->visitor_can_view_content()` has an incorrect PHPDoc (`array`).